.github/workflows/ci.yml

```diff
@@ -44,9 +44,6 @@ jobs:
       - name: Install package
         run: |
           poetry install --with test
-      - name: Run type-checking
-        run: |
-          poetry run mypy .
       - uses: codecov/codecov-action@v5
```

Comment:
I'd remove the entire `typecheck` job here if adding to `pytest`, it's just setup and teardown at this point. 😅

Reply:
oops! I didn't notice

Reply:
done!